example/src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
+openapi-boot-starter.swagger-ui.api-docs.openApiEndpoint.path = /v3/api-docs
```

Comment:
please don't use swagger word at all - its deprecated. Also ui suffix is wrong completely, as its not for ui

Reply:
is this form ok?  `openapi-boot-starter.api-docs.openApiEndpoint.path` ?

Reply:
openapi.service.endpoint I think should be fine